src/views/spectatingPlayer/SpectatingPlayer.tsx

```diff
@@ -200,14 +204,22 @@ export class SpectatingPlayer extends BaseComponent<SpectatingPlayerProps, {}> {
                 className={classNames.spectatingPlayer}
                 data-show-spectating-player={this.props.showSpectatingPlayer}
                 data-has-player-image={playerImage !== null}
+                data-image-inline={this.props.imageInline}
             >
-                {this.props.image &&
+                {this.props.image && this.props.imageInline &&
```

Comment:
I think these codes(between L209 and L222) can be  like below.
```js
{this.props.image &&
  <img
    src={playerImage}
    className={this.props.imageInline ? classNames.playerImage : classNames.spectatingPlayerImage}
    data-team={this.props.team}
   />
}
```

Reply:
Ah yes that's a lot cleaner! Thanks for the suggestion.